components/KeywordLink/KeywordLink.js

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+import HashIcon from '../../public/img/hashicon.svg';
+
+export const KeywordLink = (prop) => (
+  <div className="block py-1 md:inline md:px-5 md:first:pl-0 text-accent-500">
```

Comment:
Are the components horizontal on desktop and vertical on mobile? What does the design look like? Why is the `md:first:pl-0` needed?

Reply:
![Snímek obrazovky 2022-04-08 v 22 19 35](https://user-images.githubusercontent.com/65444975/162524132-176f99da-4f04-4e06-8bea-4bc4b82e0166.png)

Reply:
if there is no :first, then the first element will have the padding too.